cumulusci/robotframework/Salesforce.py

```diff
@@ -958,10 +968,13 @@ def salesforce_query(self, obj_name, **kwargs):
             query += "Id"
         query += " FROM {}".format(obj_name)
         where = []
-        for key, value in kwargs.items():
-            if key == "select":
-                continue
-            where.append("{} = '{}'".format(key, value))
+        if "where" in kwargs:
+            where = [kwargs["where"]]
+        else:
+            for key, value in kwargs.items():
```

Comment:
It doesn't look like this code is prepared to handle the case where someone accidentally (or on purpose...) uses both keyword arguments and the "where" keyword. Should that be spelled out in the docs and/or handled explicitly in the code here?

For example:  `Salesforce query  FirstName="Bob"  where=FirstName="James"`

In this case, `FirstName="Bob"` is completely ignored. Is that intentional, or should the keyword throw a usage error?